src/api/app/views/webui2/webui/staging/workflows/_problems_item.html.haml

```diff
@@ -1,16 +0,0 @@
-- staging_project.problems[index].each do |name, states|
```

Comment:
why did you rename this file? :confused:

Reply:
@Ana06 Since we render to partials now, i thought its more specific to call it build_problems and status_problems instead of just problems_item